rollup.config.js

```diff
@@ -34,13 +34,16 @@ const getPlugins = shouldMinify => {
     sourcemaps()
   ];
 };
+const footer = `('Auth0Client' in this) && this.console && this.console.warn && this.console.warn('Auth0Client already declared on the global namespace');
```

Comment:
I don't understand the changes to this file in context of the PR description - should this be something separate?

Reply:
This exposes the constructor on the window object (as `window.Auth0Client`)

Similar to how Marionette provides the alias `window.Mn` https://github.com/marionettejs/backbone.marionette/blob/master/rollup.config.js#L28

Reply:
I believe the idea here is to provide a warning in the browser console when there is already an `Auth0Client` symbol defined on the window object, as including our library would then try to override it.

@adamjmcgrath can you remind me of the behaviour here? Does it override it, or do we *not* override it in favour of the object that was already there?

Reply:
It doesn't override `Auth0Client` if that property already exists on the `window`, it'll raise a warning if this is the case.